---
title: 'Metagenomics and the microbiome'
date: 2199-01-01
permalink: /posts/2012/08/metagenomics/
tags:
  - microbiome
  - metabarcoding
  - metagenomics
---

## Microbiome diversity and function in human, animals, and food

### Coordinator: Rebecca Ansorge

Microbiome studies have experienced an extreme increase in the field of microbiology. Until recent years, this focussed on 16S amplicon sequencing which continues to be the go-to standard when first assessing the diversity of microbiomes and microbial communities. But since sequencing costs decreased, shotgun metagenomics of entire communities has become feasible for large-scale studies, providing much higher resolution at taxonomic and functional assessment. While conceptually both approaches have quite a lot in common, the handling of both data types differs substantially.

Over the years, excellent analyses packages and tutorials have been developed that help microbiologists to analyse and interpret their 16S amplicon microbiome data (_e.g._ Qiime2, phyloseq...). Our training modules will build upon and integrate these existing resources. Shotgun metagenomics on the other hand has been lacking behind in such standardized approaches due to i) representing a younger field, and ii) the vast complexity and scale of the data that not only provides a lot of opportunities but also major challenges to determine appropriate analyses. Here, our training will enable the participants to evaluate and determine suitable analyses for their own data rather than using a ‘one size fits all’ approach. While the detailed usage of bioinformatic tools (e.g. assemblers, read profilers, gene predicters) are often documented and easy to follow, the decision of which tool to use, how to assess suitability for the own data, and how to troubleshoot the analyses are often ignored in tutorials.

Our training module aims specifically at equipping the users with the skills and knowledge they need to independently evaluate, explore and analyse their own data. Our training module will include real datasets and those that contain low quality or failed experiments to highlight the need of suitable controls and sanity checks. The users will learn how to incorporate visualization of the data (e.g. using R or python) as checkpoints of the conducted analyses to understand their data and shed light on the “black boxes” of bioinformatic tools. 

This knowledge will also allow the users to better plan their sequencing projects, having in mind the framework of analyses already at this stage which will bring the standards of microbiome analyses to a higher level.
